frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -84,106 +101,125 @@ describe('Kubernetes resource CRUD operations', () => {
     'snapshot.storage.k8s.io~v1beta1~VolumeSnapshot',
   ]);
 
-  testObjs.forEach(({ kind, namespaced = true, humanizeKind = true }, resource) => {
-    describe(kind, () => {
-      const name = `${testName}-${_.kebabCase(kind)}`;
-
-      it(`creates the resource instance`, () => {
-        cy.visit(
-          `${namespaced ? `/k8s/ns/${testName}` : '/k8s/cluster'}/${resource}?name=${testName}`,
-        );
-        if (kind === 'Secret') {
-          listPage.clickCreateYAMLdropdownButton();
-        } else {
-          listPage.clickCreateYAMLbutton();
-        }
-        if (resourcesWithCreationForm.has(kind)) {
-          cy.byTestID('yaml-link').click();
-        }
-        // sidebar needs to be fully loaded, else it sometimes overlays the Create button
-        cy.byTestID('resource-sidebar').should('exist');
-        yamlEditor.isLoaded();
-        cy.testA11y(`YAML Editor for ${kind}: ${name}`);
-        let newContent;
-        // get, update, and set yaml editor content.
-        yamlEditor.getEditorContent().then((content) => {
-          newContent = _.defaultsDeep(
-            {},
-            { metadata: { name, labels: { [testLabel]: testName } } },
-            safeLoad(content),
+  // for debugging only, often a need to test a subset of kinds/objs
+  // ex: const testOnly = ['Pod', 'ResourceQuota'];
+  const testOnly = [];
```

Comment:
It would be better to make this some kind of env variable or param to the test rather than needing to change code. I would take this out of the PR for now.

Reply:
removed from this PR, but there is a real need to do something in this regard.  No devs wants to run all 28 resources of the CRUD test, usually only one or two.